src/pykeen/training/training_loop.py

```diff
@@ -265,9 +363,17 @@ def _train(  # noqa: C901
             If provided split each batch into sub-batches to avoid memory issues for large models / small GPUs.
         :param num_workers:
             The number of child CPU workers used for loading data. If None, data are loaded in the main process.
+        :param save_checkpoints:
+            Activate saving checkpoints.
+        :param checkpoint_path:
+            The full filepath for saving checkpoints.
+        :param checkpoint_frequency:
+            The frequency of saving checkpoints in minutes. Setting it to 0 will save a checkpoint after every epoch.
+        :param checkpoint_on_failure_file_path:
+            TODO
```

Comment:
needs docs

Reply:
also, it used to say moin. so Moin right back at ya

Reply:
https://en.wikipedia.org/wiki/Moin

Reply:
I know moin ;)